### PR TITLE
Create WebDriverConfig DTO and Test

### PR DESCRIPTION
**Pull request by N1k0l4ss**

### CODE REVIEW

File: src/main/java/executor/service/model/ProxyNetworkConfig.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package executor.service.model;
+
+import java.util.Objects;
+
+public class ProxyNetworkConfig {
+    String hostname;
```
Comment (artemkedyk) [inline src/main/java/executor/service/model/ProxyNetworkConfig.java:6]: add private modifier

File: src/test/java/executor/service/model/ProxyNetworkConfigTest.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package executor.service.model;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ProxyNetworkConfigTest {
```
Comment (artemkedyk) [inline src/test/java/executor/service/model/ProxyNetworkConfigTest.java:8]: remove this test

File: src/test/test.iml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
```
Comment (artemkedyk) [inline src/test/test.iml:3]: add this file type to the gitignore

File: src/test/java/executor/service/model/ProxyNetworkConfigTest.java (hunk 2 above)
Comment (N1k0l4ss) [reply src/test/java/executor/service/model/ProxyNetworkConfigTest.java:8]: Remove file or methods?

**Pull request closed without merging**